boards/arm/twr_ke18f/doc/index.rst

```diff
@@ -84,6 +90,28 @@ Serial Port
 The KE18 SoC has three UARTs. UART0 is configured for the console. The
 remaining UARTs are not used.
 
+Accelerometer and magnetometer
+==============================
+
+The TWR-KE18F board by default only supports polling the FXOS8700
+accelerometer and magnetometer for sensor values
+(``CONFIG_FXOS8700_TRIGGER_NONE=y``).
+
+In order to support FXOS8700 triggers (interrupts) the 0 ohm resistors
+``R47`` and and ``R57`` must be mounted on the TWR-KE18F board. The
```

Comment:
Do you also have to change the value of CONFIG_FXOS8700_TRIGGER_NONE ?

Reply:
@dbkinder You can change it, if you want to use the triggers - but you do not have to.

Reply:
This paragraph starts by saying what is needed to support triggers, so I'd think this paragraph should also mention that CONFIG_FXOS8700_TRIGGER_NONE would need to be changed to support triggers too, right?

Reply:
@dbkinder Good point. I have added that after the DTS snippet.